Ubika/ubika-cloud-protector-traffic/_meta/manifest.yml

```
@@ -0,0 +1,9 @@
+uuid: 8d024a2b-3627-4909-818d-26e1e3b2409c
+name: Ubika Cloud Protector Traffic
+slug: ubika-cloud-protector-traffic
```

As the format is related to a connector, we have some additional information to add to the format now:
- The module identifier
- The connector identifier
```suggestion
slug: ubika-cloud-protector-traffic
automation_connector_uuid: 60b3ed59-e7e4-45fa-84e8-5a52695116c1
automation_module_uuid: 0c82ee9b-f645-47f9-8e16-a689cfc246c4
```